gtsfm/scene_optimizer.py

```diff
@@ -367,17 +362,20 @@ def align_estimated_gtsfm_data(
         Updated ba_output GtsfmData object aligned to axes.
         Updated gt_pose_graph with GT poses aligned to axes.
     """
+    ba_input = ba_input.align_via_Sim3_to_poses(gt_wTi_list)
```

Comment:
from what I call, order of ops was very important for these alignment transforms. are we sure this still works out?

Reply:
yes, there are 2 alignments going on: 
1. aligning estimated poses to GT poses
2. aligning estimated poses so as to center the reconstruction

(1) was previously not happening for the saved gtsfm data. For the pose plots (which are not very usable for large datasets) (2) was happening before (1).